arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -233,18 +233,13 @@ class CpuStressorParams(CommonStressorParams):
     ] = CpuMethod.ALL
 
     def to_jobfile(self) -> str:
-        params = schema.build_object_schema(CpuStressorParams).serialize(
-            CpuStressorParams(
-                Stressors(self.stressor),
-                self.workers,
-                self.cpu_ops,
-                self.cpu_load,
-                CpuMethod(self.cpu_method),
-            )
+        return f"cpu {self.workers}\n" + params_to_jobfile(
+            {
+                "cpu-ops": self.cpu_ops,
+                "cpu-load": self.cpu_load,
+                "cpu-method": CpuMethod(self.cpu_method),
```

Comment:
Actually, if `CpuMethod(self.cpu_method)` works, here, then `self.cpu_method` must already *be* the ENUM `.value`, so simply passing `self.cpu_method` should have exactly the same effect with less obfuscation ...